src/main.py

```diff
@@ -0,0 +1,119 @@
+# (c) Open Compute Project
+# (c) Google LLC
+# (c) Igor Semenov (goshik92@gmail.com)
+# 
+# Use of this source code is governed by an MIT-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/MIT.
+
+import ocptv.output as tv
+import argparse
+import sh
+import re
+import os
+import contextlib
+from memtester_parsing import MemtesterObserver
+
+def main():
+    parser = argparse.ArgumentParser(description="OCP-compliant memtester wrapper.")
+    parser.add_argument("--mt_path", default="memtester",
+        help="Path to memtester executable")
+    parser.add_argument("--mt_args", default="",
+        help="Memtester arguments packed into a string. See `man memtester` for more detail")
+    parser.add_argument("--mt_log_filename", default=None,
+        help="Path to the file that will contain raw memtester output (both STDOUT and STDERR)")
```

Comment:
it's not clear from this that the diag wrapper here **will write** to this file (i first read this as if it were input instead of running the memtester binary)

Reply:
Attempted to improve it.

Reply:
> Path to the log file for dumping raw memtester output

yeah, this works for me; thanks